backend/app/Http/Resources/CountSessions.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+use App\Contracts\ApiResponse;
+
+class CountSessions extends JsonResource implements ApiResponse
+{
+    public function toArray($request): array
+    {
+        return [
+            'qty_sessions' => $this->resource
```

Comment:
What is qty?

Reply:
qty is short form of quantity ) 
need to fix it?

Reply:
yes, don't use acronyms and short forms, unless it is obvious)